tests/utils.js

```diff
@@ -122,3 +122,35 @@ test('Config', t => {
 
   t.deepEqual(result, expected, 'Transforms input name:value in to object');
 });
+
+test('Request Format', t => {
+  const input = {
+    'sunset-date': '',
+    'sunset-time': '',
+    'sunrise-date': '2016-06-23',
+    'sunrise-time': '00:00',
+    'service-name--text': 'bar',
+    'service-email--email--0': 'foo@test.com',
+    'service-email--email--1': 'bar@test.com',
```

Comment:
Please add what would be the output of a set of checkboxes or a multi-select, which should be an array.